packages/common/src/eips.ts

```diff
@@ -471,6 +471,22 @@ export const EIPs: EIPsDict = {
     minimumHardfork: Hardfork.London,
     requiredEIPs: [],
   },
+  6800: {
+    comment: 'Ethereum state using a unified verkle tree',
+    url: 'https://github.com/ethereum/EIPs/pull/6800',
+    status: Status.Draft,
+    minimumHardfork: Hardfork.London,
+    requiredEIPs: [],
+    gasConfig: {},
+    gasPrices: {
+      tx: {
+        v: 34300,
+        d: 'Per transaction. NOTE: Not payable on data of calls between transactions',
+      },
+    },
+    vm: {},
+    pow: {},
+  },
```

Comment:
Can you add various `experimental` notes here, so to 6800, Kaustinen and Prague (always in the `comment` field)?